### ISSUE TITLE
Condon-Shortley Phase Factor

### ISSUE THREAD

**Issue by svvelamp** [jishnub/SphericalHarmonics.jl#93]
According to the documentation for Plm (https://docs.juliahub.com/SphericalHarmonics/NjDk0/0.1.14/#SphericalHarmonics.computePlmcostheta-Tuple{Any}) and for the spherical harmonics Ylm (https://docs.juliahub.com/SphericalHarmonics/NjDk0/0.1.14/#SphericalHarmonics.computeYlm ) the C-S factor (-1)^m is _**not included**_ in the definitions. However, that's not what I am observing. For example, the formula for Y21 _**with the C-S factor**_  is given by

Y21 (theta, phi) = (-)sqrt(15/32pi) sin(2*theta) exp(i phi).

For theta=pi/3, phi=0, it gives a value of -0.3345232717786446.

Now following code produces exactly the same value:
ynm = computeYlm(pi/3,0.0,lmax=2); ynm[(2,1)]
-0.33452327177864466 - 0.0im

I have verified the same behavior for Y(1,1) as well.

Is it possible that the documentation is incorrect? Or am I missing something? [I can think of another possibility, the branch used in defining (1-x^2)^{m/2}) in the definition of Pnm.]

Could you please clarify?

Thanks,
Sanjay Velamparambil

**Comment by svvelamp** [comment]
One follow-up: in the cited reference by [Limpanuparb and Milthorpe ](http://arxiv.org/abs/1410.1748), the definition of Plm does include a factor of (-1)^m.

**Comment by jishnub** [comment]
Yes, the docs were incorrect there. The phase factor is, in fact, included. The docs are [updated now](https://jishnub.github.io/SphericalHarmonics.jl/dev/#SphericalHarmonics.computePlmcostheta-Tuple{Any})

**Comment by svvelamp** [comment]
thank you!
________________________________
